modules/@apostrophecms/login/index.js

```diff
@@ -817,6 +821,17 @@ module.exports = {
             return next();
           }
         }
+      },
+      addLoggedInCookie: {
+        before: '@apostrophecms/i18n',
```

Comment:
needed before `'@apostrophecms/i18n'`, like the other middleware from this module?